src/kOS/Module/KOSNameTag.cs

```diff
@@ -37,10 +38,31 @@ public void PopupNameTagChanger()
                     return;
                 }
             }
+            // Make a new instance of typingWindow, replacing the existing one if there was one:
+            KOSNameTagWindow oldTypingWindow = gameObject.GetComponent<KOSNameTagWindow>();
+            if (oldTypingWindow != null)
+            Destroy(oldTypingWindow);
```

Comment:
The above change wasn't part of the fix.  I just noticed the fact that i was orphaning windows for the GC to collect when I could have been explicitly telling Unity they were going away, and probably should have been since Unity doesn't really notice orphaning (it's C++, not C#).